src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -508,7 +508,11 @@ function_call_harness_generatort::implt::declare_arguments(
         associated_array_size_argument == parameter_name_to_argument_name.end())
       {
         throw invalid_command_line_argument_exceptiont{
-          "associated array size is not there",
+          "could not find parameter to associate the array size of array \"" +
+            id2string(parameter_name) + "\" (Expected parameter: \"" +
+            id2string(associated_array_size_parameter) + "\" on function \"" +
+            id2string(function_to_call.display_name()) + "\" in " +
+            function_to_call.location.as_string() + ")",
```

Comment:
Yeah this is probably for the better. For a second I was wondering whether the redundancy of repeating the function name would be a problem, but honestly it shouldn’t hurt and might help when running this from a script.

Reply:
Where is the function repeated? (for convenience I attached the new error message to the PR description)